Analytics/resources/export_data/export_data.py

```diff
@@ -153,91 +122,123 @@ def get_location(self, location_id: int) -> db.Model:
 
         return None
 
-    def frame_data(self, column_names: [str], entries: [Any]) -> pd.DataFrame:
-        """
-        Create Pandas DataFrame
-        :param column_names: List of column names
-        :param entries: List of values
-        :return: Pandas DataFrame
-        """
-        if not column_names or not entries:
-            return None
-
-        data_frame = pd.DataFrame.from_records(entries, columns=column_names)
-        return data_frame
-
-    def write_to_csv(self, data_frame: pd.DataFrame, file_name: str, extension: str) -> bool:
+    @staticmethod
+    def write_to_csv(data_frame: pd.DataFrame, file_name: str, directory: os.path, extension: str) -> bool:
         """
         Write data to csv file for export
+        :param directory: Absolute path to create file
         :param data_frame: Pandas DataFrame of data
         :param file_name: Name of the file to save csv to
         :param extension: File extension
         :return: If exporting data to file is successful True is returned, otherwise False
         """
-        directory = os.path.dirname(os.path.realpath(__file__)) + "/"
+        file = os.path.join(directory, file_name + "." + extension)
         try:
-            data_frame.to_csv(directory + file_name + "." + extension, sep=',', encoding='utf-8')
+            data_frame.to_csv(file, sep=',', encoding='utf-8', index=False)
             return True
-        except Exception:
+        except Exception as e:
```

Comment:
Shouldn't catch the exception like this, rather catch expected exceptions, then fail otherwise?